x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.scss

```diff
@@ -0,0 +1,14 @@
+// Add truncation to heath status
+
+.actionsInstanceList__health {
+  width: 100%;
+
+  .euiFlexItem:last-of-type {
+    min-width: 0;
```

Comment:
There's an EUI mixin for truncating text that you can likely use instead:

https://github.com/elastic/eui/blob/4248003ee02efda5ab56dd2556d6821bbf4b0fea/src/global_styling/mixins/_typography.scss#L108-L115